exporter/collector/logs.go

```diff
@@ -43,20 +62,149 @@ func (l *LogsExporter) Shutdown(ctx context.Context) error {
 }
 
 func (l *LogsExporter) PushLogs(ctx context.Context, ld pdata.Logs) error {
+	mapper := &metricMapper{} // Refactor metricMapper to map MRs for logging?
+	logPushErrors := []error{}
 	for i := 0; i < ld.ResourceLogs().Len(); i++ {
 		rl := ld.ResourceLogs().At(i)
-		mapper := &metricMapper{} // Refactor metricMapper to map MRs for logging?
 		mr, _ := mapper.resourceToMonitoredResource(rl.Resource())
 		for j := 0; j < rl.InstrumentationLibraryLogs().Len(); j++ {
 			ill := rl.InstrumentationLibraryLogs().At(j)
 			for k := 0; k < ill.LogRecords().Len(); k++ {
 				log := ill.LogRecords().At(k)
-				l.logger.Log(logging.Entry{
-					Resource: mr,
-					Payload:  log.Body().AsString(),
-				})
+				entry, err := l.logToEntry(log, make(map[string]interface{}), mr)
+				if err != nil {
+					logPushErrors = append(logPushErrors, err)
+				} else {
+					l.logger.Log(entry)
+				}
 			}
 		}
 	}
+	if len(logPushErrors) > 0 {
+		return multierr.Combine(logPushErrors...)
+	}
 	return nil
 }
+
+func (l *LogsExporter) logToEntry(
+	log pdata.LogRecord,
+	attributes map[string]interface{},
+	mr *monitoredres.MonitoredResource,
+) (logging.Entry, error) {
+	entry := logging.Entry{
+		Timestamp: log.Timestamp().AsTime(),
+		Severity:  logging.Severity(log.SeverityNumber()),
+		Trace:     log.TraceID().HexString(),
+		SpanID:    log.SpanID().HexString(),
```

Comment:
I couldn't get logs to send through to the cloud logging dashboard with these 4 fields. Maybe we disable them for now. We will have to do more processing on Severity and Timestamp anyway

Reply:
Fixed in [72b3690](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/344/commits/72b3690d4a49970b7eaffdd10072dab376d4c72e)